docs/api/extras/helpers/ArrowHelper.html

```diff
@@ -80,7 +80,7 @@
 		Sets the length of the arrowhelper.
 		</div>
 
-		<h3>[method:null setDirection]([page:vector3 dir])</h3>
+		<h3>[method:null setDirection]([page:Vector3 dir])</h3>
 		<div>
 		dir -- The desired direction in euler format.
```

Comment:
dir -- The desired direction. It must be normalized ( have unit length ).

It is not a Euler vector.